src/helpers/schema/schema.helper.ts

```diff
@@ -26,41 +28,39 @@ export class SchemaHelper {
       if (schema.default != undefined) {
         return schema.default;
       }
-      if (schema.type == SchemaTypes.ENUM && schema.enum != undefined && schema.enum != null) {
+      if (schema.type == SchemaTypes.ENUM && schema.enum != undefined) {
         if (schema.enum.length == 0) {
           return "[" + schema.name + "]";
         }
-        var random = Math.floor((Math.random() * schema.enum.length));
+        const random = Math.floor((Math.random() * schema.enum.length));
         return schema.enum[random];
       } else if (schema.type == SchemaTypes.BOOLEAN) {
         return true;
       } else if (schema.type == SchemaTypes.INTEGER) {
-        var random = 13;
-        return random;
+        return 13;
       } else if (schema.type == SchemaTypes.NUMBER) {
-        var random = 13.6;
-        return random;
+        return 13.6;
       } else if (schema.type == SchemaTypes.STRING) {
         return "string";
       } else if (schema.type == SchemaTypes.ARRAY) {
-        var array: Array<any> = [];
+        const array: Array<any> = [];
         array.push(SchemaHelper.generateExample(schema.items, undefined, objectStack, rootObject));
         return array;
       } else if (schema.type == SchemaTypes.OBJECT) {
-        var object: any = {};
+        const object: any = {};
         if (schema.allOf != undefined) {
           schema.allOf.forEach(superSchema => {
-            var superObject = SchemaHelper.generateExample(superSchema, fieldName, objectStack.slice(), rootObject);
-            for (var field in superObject) {
+            const superObject = SchemaHelper.generateExample(superSchema, fieldName, objectStack.slice(), rootObject);
+            for (let field in superObject) {
               object[field] = superObject[field];
             }
           });
         }
-        for (var field in schema.properties) {
-          var property = schema.properties[field];
-          var name = field;
-          var jsonName = SchemaHelper.resolveReference('mappings.json.name', property);
-          var jsonIgnore = SchemaHelper.resolveReference('mappings.json.ignore', property);
+        for (const field in schema.properties) {
+          const property = schema.properties[ field ];
```

Comment:
```suggestion
          const property = schema.properties[field];
```
mostly we don't give spaces for accessing the property of an object/array 🙂